.github/bin/run-clang-tidy.sh

```diff
@@ -13,60 +13,79 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Script to run clang-tidy on files in a bazel project while caching the
+# results as clang-tidy can be pretty slow. The clang-tidy output messages
+# are content-addressed in a hash(cc-file-content) cache file.
+
 set -u
 
+readonly PROJECT_NAME=verible           # Prefix for directory in ${TMPDIR}
+readonly EXIT_CODE_ON_WARNINGS=1        # Should fail on warnings found ?
+readonly CLANG_CONFIG_FILE=.clang-tidy  # config file used
+
 if [ ! -f WORKSPACE ]; then
   echo "This script needs to be invoked in the root of the bazel project"
   exit 1
 fi
 
-# The caller can supply additional clang-tidy flags that are passed
-# to it as-is.
+# The caller can supply additional clang-tidy flags that are passed as-is
 # Example:
-# .github/bin/run-clang-tidy.sh --checks="-*,modernize-use-equals-default" --fix
-readonly ADDITIONAL_TIDY_OPTIONS=("$@")
+#   run-clang-tidy.sh --checks="-*,modernize-use-equals-default" --fix
+readonly ADDITIONAL_TIDY_OPTIONS="$@"
```

Comment:
FYI, I switched that back as regular scalar, not array. I couldn't get the quoting to work around line 170.